contributors/design-proposals/mount-options.md

```diff
@@ -0,0 +1,76 @@
+# Mount options for mountable volume types
+
+## Goal
+
+Enable Kubernetes admins to specify mount options with mountable volumes
+such as  - `nfs`, `glusterfs` or `aws-ebs` etc.
+
+## Motivation
+
+We currently support network filesystems: NFS, Glusterfs, Ceph FS, SMB (Azure file), Quobytes, and local filesystems such as ext[3|4] and XFS.
+
+Mount time options that are operationally important and have no security implications should be suppported. Examples are NFS's TCP mode, versions, lock mode, caching mode; Glusterfs's caching mode; SMB's version, locking, id mapping; and more.
+
+## Design
+
+Mount options can be specified as `mountOption`  as part of PV. As an example:
+
+``` yaml
+ apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0003
+  spec:
+    capacity:
+      storage: 5Gi
+    accessModes:
+      - ReadWriteOnce
+    persistentVolumeReclaimPolicy: Recycle
+    mountOption: ["hard","nolock","nfsvers=3"]
```

Comment:
Talking with Hemant about this offline yesterday, i dont think the use cases for mountOptions in storageclass is large enough to warrant adding the function or a field now and should nix it from the proposal.  It would be easier to solve mount options in storage class when someone needs it and has a good reason.